src/Models/LoginModel.php

```diff
@@ -39,7 +39,7 @@ class LoginModel extends BaseModel
         'id_type'    => 'required',
         'identifier' => 'permit_empty|string',
         'user_agent' => 'permit_empty|string',
-        'user_id'    => 'permit_empty|integer',
+        'user_id'    => 'permit_empty',
```

Comment:
I think this is fine.
Since this model only logs login (attempt), it does not seem necessary to be very strict with validation.

Since user id can be letters instead of a number, `integer` is obviously wrong.

Reply:
Oukayyy, ahh i have a question bro, is JWT Auth Setting key has sync with CI Setting library ? maybe i want try the scenario of saving key on database

Reply:
No, at the moment. See the source code, it uses `config()` instead of `settings()`.